sources/packages/backend/apps/api/src/route-controllers/designation-agreement/designation-agreement.aest.controller.ts

```diff
@@ -151,15 +149,6 @@ export class DesignationAgreementAESTController extends BaseController {
         );
       }
     }
-    const submissionResult = await this.formService.dryRunSubmission(
```

Comment:
When the dry-run validation is removed, we must use class-validators to impose the validation.